HLTrigger/Configuration/python/customizeHLTforAlpaka.py

```diff
@@ -1,5 +1,255 @@
+                    0.2,
+                    0.2,
+                    0.2
+                  )
+                )
+              )
+            ),
+            pfClusterBuilder = cms.PSet(
+              maxIterations = cms.uint32(5),
+              minFracTot = cms.double(1e-20),
+              minFractionToKeep = cms.double(1e-07),
+              excludeOtherSeeds = cms.bool(True),
+              showerSigma = cms.double(10),
+              stoppingTolerance = cms.double(1e-08),
+              recHitEnergyNorms = cms.VPSet(
+                cms.PSet(
+                  detector = cms.string('HCAL_BARREL1'),
+                  recHitEnergyNorm = cms.vdouble(
+                    0.1,
+                    0.2,
+                    0.3,
+                    0.3
+                  )
+                ),
+                cms.PSet(
+                  detector = cms.string('HCAL_ENDCAP'),
+                  recHitEnergyNorm = cms.vdouble(
+                    0.1,
+                    0.2,
+                    0.2,
+                    0.2,
+                    0.2,
+                    0.2,
+                    0.2
+                  )
+                )
+              ),
+              positionCalc = cms.PSet(
+                minFractionInCalc = cms.double(1e-09),
+                minAllowedNormalization = cms.double(1e-09)
+              ),
+              timeResolutionCalcBarrel = cms.PSet(
+                corrTermLowE = cms.double(0),
+                threshLowE = cms.double(6),
+                noiseTerm = cms.double(21.86),
+                constantTermLowE = cms.double(4.24),
+                noiseTermLowE = cms.double(8),
+                threshHighE = cms.double(15),
+                constantTerm = cms.double(2.82)
+              ),
+              timeResolutionCalcEndcap = cms.PSet(
+                corrTermLowE = cms.double(0),
+                threshLowE = cms.double(6),
+                noiseTerm = cms.double(21.86),
+                constantTermLowE = cms.double(4.24),
+                noiseTermLowE = cms.double(8),
+                threshHighE = cms.double(15),
+                constantTerm = cms.double(2.82)
+              )
+            ),
+            )
+
+    process.hltPFClusterSoAProducer = cms.EDProducer("PFClusterSoAProducer@alpaka",
+            pfRecHits = cms.InputTag("hltPFRecHitSoAProducerHCAL"),
+            topology = cms.ESInputTag("hltPFRecHitHCALTopologyESProducer:"),
+            pfClusterParams = cms.ESInputTag("hltPFClusterParamsESProducer:"),
+            synchronise = cms.bool(False)
+            )
+
+    process.hltPFClusterSoAProducerCPUSerial = makeSerialClone(process.hltPFClusterSoAProducer,
+            pfRecHits = cms.InputTag("hltPFRecHitSoAProducerHCALCPUSerial"),
+            )
+
+    process.hltLegacyPFClusterProducer = cms.EDProducer("LegacyPFClusterProducer",
+            src = cms.InputTag("hltPFClusterSoAProducer"),
+            pfClusterParams = cms.ESInputTag("pfClusterParamsESProducer:"),
```

Comment:
Should this be `hltPFClusterParamsESProducer:` instead ? (I guess it does not matter, since the tests passed ? From a quick look, it seems that `pfClusterParams` is not really used inside `LegacyPFClusterProducer`)